src/bp-blogs/classes/class-bp-blogs-template.php

```diff
@@ -176,8 +174,6 @@ public function __construct( ...$args ) {
 		// Set the current blog count.
 		if ( empty( $r['max'] ) || ( $r['max'] >= (int) $blog_count ) ) {
 			$this->blog_count = (int) $blog_count;
-		} else {
-			$this->blog_count = (int) $max;
```

Comment:
Why do you remove the default `blog_count` value ?

Reply:
If you look at the code, you will see that the `$max` variable does not exist. This code defaults to 0 since `(int) null === 0`. Since my previous changes, the `$blog_count` property already defaults to 0, so this is unnecessary.

Reply:
Source: https://github.com/buddypress/buddypress/blob/989ed99592aea76bafb0ffac9a5b32520603974c/src/bp-blogs/classes/class-bp-blogs-template.php#L180

Reply:
As explained in my previous comment, we should make sure $max is defined instead of removing it. Eg:

`$max = (int) wp_unslash( $r['max'] );`